packages/utils/dropdown/resourceDropdown.scss

```diff
@@ -14,3 +14,8 @@
 .mcgms-resourceDropdown__container > button {
   width: 100%;
 }
+
+.mcgms-mutedText__container {
+  padding-left: 24px;
+  width: 100%;
+}
```

Comment:
why ? I don't see much diff because of it. 
Also, different resources will have different badge length, this CSS might work for particular case, but can affect other places.

Reply:
`24px` is working because we are showing `S` badge right now... this will not work for any other badge. We can't make generic component to serve a specific usecase... try to make it generic.

Reply:
it is okay to have it aligned with the badge instead of the text... that will be doable in the ResourceDropdown itself and will be same for all the components using the dropdown...

Reply:
as I remember @cloudbehl asked me to add some padding to the namespace and bring it below the name.
if we don't want we can remove displaying of namespace completely as the user is selecting/seeing which namespace is selected and for obvious reason, the secrets will belong to that only.

Reply:
we added showing namespace because we started to have secrets from multiple namespaces, but since we are not going down that road, I think we should stick to what we have in our current master.

![image](https://user-images.githubusercontent.com/22158580/177532418-2b6beae8-c815-41d7-9753-8af7ab2b1367.png)

Reply:
we can remove the namespace muted text... we will fix the css issue later...

Reply:
cool removing it. how it looks after removing 

![image](https://user-images.githubusercontent.com/22158580/177593394-19efc316-2fa4-47a3-957a-ca401577c325.png)

Reply:
I meant we can remove secondary text itself... u are right: `we added showing namespace because we started to have secrets from multiple namespaces, but since we are not going down that road, I think we should stick to what we have in our current master.`

Reply:
removed :+1: 

![image](https://user-images.githubusercontent.com/22158580/177597581-84f75f7c-7d40-408b-aaf5-40162f99715d.png)